### DISCUSSION TITLE
How can I install this addon?

### DISCUSSION THREAD

**Discussion by SirLouen** [guerrerotook/securitas-direct-new-api#77] [Q&A] [answered]
What is the protocol to install this?
I can't find any docs for this type of addon installation
I tried to use the github as a repository in the addons part of HA but it doesn't seem to work

**Comment by siom7** [comment, marked as answer]
Hello, It's not an Add-On but a Custom Component.
You need to use HACS (https://www.wundertech.net/how-to-install-hacs-on-home-assistant/) and after that add URL (https://github.com/guerrerotook/securitas-direct-new-api) repository to install and configure it.

**Comment by danigd82** [comment]
Buenas! Disculpad mi ignorancia, pero carezco de conocimientos de programación como para adentrarme en esto... Resulta que me gustaría cambiar el icono de Armado en Casa, por el de la LUNA en el panel de control, ya que mi señora esposa siempre se confunde... No encuentro la forma de hacerlo.. se que es algo trivial, pero me ahorraría tener que estar desconectando la alarma y escuchando a mi señora esposa.
Gracias!
